src/tests/cli/firewall-cmd.at

```diff
@@ -1175,6 +1175,12 @@ FWD_START_TEST([direct])
 FWD_END_TEST([-e '/ERROR: INVALID_TABLE:/d' dnl
               -e '/WARNING: NOT_ENABLED: chain/d'])
 
+FWD_START_TEST([direct zone])
+    AT_KEYWORDS(direct zone)
+    
+    FWD_CHECK([firewall-cmd --zone=public --permanent --direct --add-rule ipv4 nat OUTPUT 1 -p tcp --dport 8443 -j DNAT --to-port 9443], 2, ignore,ignore)
```

Comment:
Please also check this combination of arguments omitting the `--permanent` flag. That is, so it would attempt to manipulate the runtime configuration.